core/frontend/src/main/scala/io/udash/bindings/modifiers/DOMManipulator.scala

```diff
@@ -0,0 +1,39 @@
+package io.udash.bindings.modifiers
+
+import org.scalajs.dom.{Element, Node}
+
+private[bindings]
+trait DOMManipulator {
+  type ReplaceMethod = (Node, Seq[Node], Seq[Node]) => Boolean
+  type InsertMethod = (Node, Node, Seq[Node]) => Boolean
```

Comment:
I cannot see a reason for using type members here. The types are not overridden anywhere, I'm not sure they even should be and if these contracts were just a part of the method signatures in this trait, the arguments would have meaningful names.